CHANGELOG.md

```diff
@@ -14,11 +14,18 @@
 - Add long-distance restrictions via `<<` and `>>` operators. #943, #969
 - Fix relative pathing for `mkdocstring-python=>1.9.1`. #967, #968
 - Clean up old `TableChain.join` call in mixin delete. #982
+- Add pytests for position pipeline, various `test_mode` exceptions #966
+- Migrate dependencies from `environment.yml`s to `pyproject.toml` #966
```

Comment:
I would request this not happen. This negates the benefits of having conda solve the environment.